code/modules/mob/living/carbon/xenomorph/castes/praetorian/abilities_praetorian.dm

```diff
@@ -467,3 +467,551 @@ GLOBAL_LIST_INIT(acid_spray_hit, typecacheof(list(/obj/structure/barricade, /obj
+	SIGNAL_HANDLER
+	if(!new_dir || new_dir == old_dir)
+		return
+	owner.overlays -= held_appearance
+	var/matrix/new_transform = held_appearance.transform
+	switch(old_dir)
+		if(NORTH)
+			new_transform.Translate(-15, -12)
+		if(SOUTH)
+			new_transform.Translate(-15, 12)
+		if(EAST)
+			new_transform.Translate(-35, 0)
+		if(WEST)
+			new_transform.Translate(5, 0)
+	switch(new_dir)
+		if(NORTH)
+			new_transform.Translate(15, 12)
+		if(SOUTH)
+			new_transform.Translate(15, -12)
+		if(EAST)
+			new_transform.Translate(35, 0)
+		if(WEST)
+			new_transform.Translate(-5, 0)
+	held_appearance.transform = new_transform
+	owner.overlays += held_appearance
+
+// ***************************************
+// *********** Tail Lash
+// ***************************************
+/datum/action/ability/activable/xeno/tail_lash
+	name = "Tail Lash"
+	action_icon_state = "tail_lash"
+	action_icon = 'icons/Xeno/actions/praetorian.dmi'
+	desc = "Knock back humans that are in front of you."
+	ability_cost = 50
+	cooldown_duration = 13 SECONDS
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOABILITY_TAIL_LASH,
+	)
+
+/datum/action/ability/activable/xeno/tail_lash/use_ability(atom/target)
+	xeno_owner.face_atom(target)
+
+	var/turf/lower_left
+	var/turf/upper_right
+	switch(xeno_owner.dir)
+		if(NORTH)
+			lower_left = locate(xeno_owner.x - 1, xeno_owner.y + 1, xeno_owner.z)
+			upper_right = locate(xeno_owner.x + 1, xeno_owner.y + 2, xeno_owner.z)
+		if(SOUTH)
+			lower_left = locate(xeno_owner.x - 1, xeno_owner.y - 2, xeno_owner.z)
+			upper_right = locate(xeno_owner.x + 1, xeno_owner.y - 1, xeno_owner.z)
+		if(WEST)
+			lower_left = locate(xeno_owner.x - 2, xeno_owner.y - 1, xeno_owner.z)
+			upper_right = locate(xeno_owner.x - 1, xeno_owner.y + 1, xeno_owner.z)
+		if(EAST)
+			lower_left = locate(xeno_owner.x + 1, xeno_owner.y - 1, xeno_owner.z)
+			upper_right = locate(xeno_owner.x + 2, xeno_owner.y + 1, xeno_owner.z)
+
+	for(var/turf/affected_tile in block(lower_left, upper_right))
+		affected_tile.Shake(duration = 0.1 SECONDS)
+		for(var/atom/movable/affected AS in affected_tile)
+			if(!ishuman(affected))
+				continue
+			if(affected.move_resist >= MOVE_FORCE_OVERPOWERING)
+				continue
+			var/mob/living/carbon/human/affected_human = affected
+			if(affected_human.stat == DEAD)
+				continue
+			affected_human.Paralyze(0.1 SECONDS)
+			affected_human.apply_effect(0.1 SECONDS, WEAKEN)
+			var/throwlocation = affected_human.loc
+			for(var/x in 1 to 2)
+				throwlocation = get_step(throwlocation, owner.dir)
+			affected_human.throw_at(throwlocation, 2, 1, owner, TRUE)
+			// Preventing this from infinitely being used as a cost-free throw against the crit.
+			if(affected_human.stat == UNCONSCIOUS)
+				RegisterSignal(affected_human, COMSIG_MOVABLE_MOVED, PROC_REF(on_movement_while_thrown))
+				RegisterSignal(affected_human, COMSIG_MOVABLE_POST_THROW, PROC_REF(on_throw_end))
```

Comment:
again, I really am not a fan of adding this snowflake when no other xeno abilities do it. Either they all should, or none of them.